src/input.js

```diff
@@ -105,6 +107,7 @@ class Input extends Component {
         aria-expanded={isOpen}
         autoComplete="off"
         value={(inputValue === null ? selectedItemValue : inputValue) || ''}
+        autoFocus={this.menu}
```

Comment:
Interesting! What does this do? Should probably pass this to the boolean constructor to make this more explicit, but what's it for?